gm4_vecto_shamir/data/gm4_vecto_shamir/functions/vehicle/check.mcfunction

```diff
@@ -0,0 +1,5 @@
+#run from vehicle/spawn_(boat/minecart) and vehicle/check
+
+execute as @e[tag=gm4_ml_vehicle] at @a[tag=gm4_ml_invehicle,predicate=gm4_vecto_shamir:not_in_vehicle] if score @s gm4_ml_vecto_id = @p[tag=gm4_ml_invehicle,predicate=gm4_vecto_shamir:not_in_vehicle,distance=..1,sort=nearest] gm4_ml_vecto_id run function gm4_vecto_shamir:vehicle/remove
```

Comment:
`type=#vecto:vehicles` should be added in the `@e` selector, in which `vecto:vehicles` contains boats and minecarts

Reply:
`@p` already sorts by nearest, so the `sort=nearest` is unnecessary